src/export/ExportUtils.h

```diff
@@ -36,5 +39,6 @@ class ExportUtils final
    static void InitProgress(std::unique_ptr<BasicUI::ProgressDialog> &pDialog,
          const wxFileNameWrapper &title, const TranslatableString &message);
 
+   static TrackIterRange<const WaveTrack> FindExportWaveTracks(const TrackList& tracks, bool selectedOnly);
```

Comment:
Do you understand the difference between `SampleTrack` and `WaveTrack`?  The first is an abstract base class of the second.  (And `WritableSampleTrack` is between them.)

I think these utilities would be better defined in terms of the more general interface.  We don't yet have other concrete subclasses of `SampleTrack` but it may be useful later.

`SampleTrack` does not expose the notion of a clip within the track, but I don't see that you require knowledge of subdivision into clips.